CaptchaBuilder.php

```diff
@@ -128,19 +133,15 @@ public function setInterpolation($interpolate = true)
      */
     public $tempDir = 'temp/';
 
-    public function __construct($phrase = null, PhraseBuilderInterface $builder = null)
```

Comment:
You're changing the constructor and disallow passing phrase to the builder, why?